apps/minifront/src/components/extension-unavailable.tsx

```diff
@@ -0,0 +1,29 @@
+import { SplashPage } from '@penumbra-zone/ui';
+import { HeadTag } from './metadata/head-tag';
+
+const NODE_STATUS_PAGE_URL =
+  window.location.hostname === 'localhost' ? 'http://localhost:5174' : '/';
```

Comment:
this probably needs a .env and webpack define

Reply:
actually, it seems to be the same as the grpc endpoint uri

Reply:
this is tricky... we'd want to define the port number in .env, you mean, right? that would mean putting `5174` in `node-status`'s `.env` file, and then importing it from minifront? (Cause I assume we'd also want to use the `.env` value in `node-status`'s `package.json` `dev` script